bincomp/Program.cs

```diff
@@ -28,10 +28,8 @@ private static void Main(string[] args)
 			Console.WriteLine($"Exported Symbols: {symbolsFilename}");
 
 			Console.WriteLine($"Cleaning up for a fresh build attempt.");
-			File.Delete(basePatchJson);
-			File.Delete(symbolsFilename);
 
-				// ReSharper disable InconsistentNaming
+			// ReSharper disable InconsistentNaming
 			// ReSharper disable JoinDeclarationAndInitializer
```

Comment:
I'm not using ReSharper at the moment, are you? If not, let's kill these.

Reply:
I do use resharper